packages/slice-machine/tests/lib/mock/Slice.test.ts

```diff
@@ -1,7 +1,7 @@
 import "@testing-library/jest-dom";
 import { WidgetTypes } from "@prismicio/types-internal/lib/customtypes/widgets";
 import { SlicesTypes } from "@prismicio/types-internal/lib/customtypes/widgets/slices";
-import { SliceSM, Slices } from "@slicemachine/core/build/models";
+import { SliceSM } from "@slicemachine/core/build/models";
 import { isRight } from "fp-ts/lib/Either";
 import MockSlice from "../../../lib/mock/Slice";
 // import allFieldSliceModel from "../../../tests/__mocks__/sliceModel";
```

Comment:
I'll need to add this test case back in

Reply:
which case?

Reply:
way down at the bottom, for allFieldSliceModel